### PR TITLE
Add Keane as blog author

### PR DESCRIPTION
**Pull request by keanerobinson**

### CODE REVIEW

**Comment by BenFradet** [comment]
@keanerobinson ^^

**Comment by keanerobinson** [comment]
hey @BenFradet made the changes to author.yml as required

File: _data/authors.yml [hunk 1]
```diff
@@ -206,3 +206,10 @@
   avatar: /assets/img/team/Anthony.jpg
   spiel: |
       <a href="/blog/authors/anthony/">Anthony</a> is the Digital Marketing Manager at Snowplow. You can find him on <a href="https://twitter.com/Mandelliant">Twitter</a>, <a href="https://www.linkedin.com/in/amandelli/">LinkedIn</a>.
+
+ name: Keane
```
Comment (BenFradet) [inline _data/authors.yml]: you need to add a dash:

```yaml
  - name: Keane
```

like Anthony above

**Comment by keanerobinson** [comment]
cheers Ben - done.

**Comment by BenFradet** [comment]
cool 👍 

@yalisassoon could we this in?

**Pull request merged**